src/com/projectkorra/projectkorra/Element.java

```diff
@@ -114,13 +120,53 @@ public String getPrefix() {
 	}
 
 	public ChatColor getColor() {
-		final String color = this.plugin.getName().equalsIgnoreCase("ProjectKorra") ? ConfigManager.languageConfig.get().getString("Chat.Colors." + this.name) : this.plugin.getConfig().getString("Chat.Colors." + this.name);
-		return color != null ? ChatColor.valueOf(color) : ChatColor.WHITE;
+		if (this.color == null) {
+			FileConfiguration config = this.plugin.getName().equalsIgnoreCase("ProjectKorra") ? ConfigManager.languageConfig.get() : this.plugin.getConfig();
+			String key = "Chat.Colors." + this.name;
+			String value = config.getString(key);
+
+			if (value == null && this instanceof SubElement && !(this instanceof MultiSubElement)) {
+				this.color = ((SubElement) this).parentElement.getSubColor();

```

Comment:
This is because subelements having individual colors is very new. If a color is not defined for them, it will then fall back to using the default subelement color defined in the parent element like before